MediaBrowser.Model/Lyrics/TimeTag.cs

```diff
@@ -0,0 +1,28 @@
+namespace MediaBrowser.Model.Lyrics;
+
+/// <summary>
+/// TimeTag model.
+/// </summary>
+public class TimeTag
```

Comment:
Should be more descriptive, at least add the "lyric" keyword to it.

```suggestion
public class LyricLineTimeTag
```